File: MercaderSG/Comercial/NotaVenta/UltimaNVRepor.vb
Imports Negocios
Imports Servicios
Imports Entidades
Imports Excepciones
Imports CrystalDecisions.CrystalReports
Imports CrystalDecisions.CrystalReports.Engine
Public Class UltimaNVRepor

    Private Sub ReportesNV_Load(sender As System.Object, e As System.EventArgs) Handles MyBase.Load
        Me.Text = My.Resources.ArchivoIdioma.UltimaNotaVentaFrm

        Dim NVDS As New GeneralDS

        NotaVentaRN.CargarUltimaNotaVenta(NVDS)

        Dim Reporte As New NotaVentaRP
        Reporte.SetDataSource(NVDS)
        UltimaNotaRPV.ReportSource = Reporte
    End Sub

    Private Sub UltimaNVRepor_KeyDown(sender As System.Object, e As System.Windows.Forms.KeyEventArgs) Handles MyBase.KeyDown
        If e.KeyCode = Keys.Escape Then
            Me.Close()
        End If
    End Sub
End Class

File: MercaderSG/Comercial/NotaVenta/UltimaNVRepor.cs
using System;
using System.Windows.Forms;
using Negocios;

namespace MercaderSG
{
    public partial class UltimaNVRepor
    {
        public UltimaNVRepor()
        {
            InitializeComponent();
        }

        private void ReportesNV_Load(object sender, EventArgs e)
        {
            Text = My.Resources.ArchivoIdioma.UltimaNotaVentaFrm;
            var NVDS = new GeneralDS();
            NotaVentaRN.CargarUltimaNotaVenta(NVDS);
            var Reporte = new NotaVentaRP();
            Reporte.SetDataSource(NVDS);
            UltimaNotaRPV.ReportSource = Reporte;
        }

        private void UltimaNVRepor_KeyDown(object sender, KeyEventArgs e)
        {
            if (e.KeyCode == Keys.Escape)
            {
                Close();
            }
        }
    }
}